source/tutorials/longitudinal.rst

```diff
@@ -115,6 +115,50 @@ Second, control charts display the mean value of "metric" at each "state". The f
 This visualizer currently only generates control charts, which are a useful **qualitative** approach for visually identifying abnormal time points; accompanying statistical tests may be added in future releases.
 
 
+First differencing to track rate of change
+------------------------------------------
+Another way to view time series data is by assessing how the rate of change differs over time. We can do this through calculating first differences, which is the magnitude of change between successive time points. If :math:`Y_\text{t}` is the value of metric :math:`Y` at time :math:`t`, the first difference at time :math:`t`, :math:`{\Delta}Y_\text{t} = Y_\text{t} - Y_\text{t-1}`. This transformation is performed in the ``first-differences`` method in ``q2-longitudinal``.
+
+.. command-block::
+
+   qiime longitudinal first-differences \
+     --m-metadata-file ecam-sample-metadata.tsv \
+     --m-metadata-file shannon.qza \
+     --p-state-column month \
+     --p-metric shannon \
+     --o-first-differences shannon-first-differences \
```

Comment:
Can you include the `.qza` file extension on the output? Though the extension is not necessary, we include it in the tutorials to make it clear what exactly is being created (without an extension, you can't tell if it is a `.qza`, `.qzv`, a directory, or Something Else :tm:).

Reply:
👍